src/test/java/io/jenkins/plugins/artifact_manager_jclouds/s3/LocalStackIntegrationTest.java

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
Yes I created another test because first I had some issues with Minio so I was testing with another.
Finally figured out the problem. I leaved this new test but can be removed if it's real pain to have more tests/